### PR TITLE
Create matrix.txt

### PR DESCRIPTION
**Pull request by rajeshgheware**
Added matrix function

### CODE REVIEW

**Review by brainupgrade-in — CHANGES_REQUESTED** [review]
add description

File: matrix.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+matrix
```
Comment (brainupgrade-in) [inline matrix.txt]: Is this main function?

**Comment by rajeshgheware** [comment]
Description added

**Review by brainupgrade-in — CHANGES_REQUESTED** [review]
see the review comments and respond

**Review by brainupgrade-in — CHANGES_REQUESTED** [review]
Add some more description

**Review by rajeshgheware — COMMENTED** [review]
Done

**Review by brainupgrade-in — APPROVED** [review]
looks good now

**Pull request merged**